tests/tests.js

```diff
@@ -64,4 +63,24 @@ describe('Initialization', function() {
 
   });
 
+  describe('Viewer', function() {
+    const WIDTH = 500;
+    const HEIGHT = 300;
+    const canvas = document.createElement('canvas');
+    canvas.width = WIDTH;
+    canvas.height = HEIGHT;
+    const viewer = new ROS3D.Viewer({
+      canvas: canvas,
+      height: HEIGHT,
+      width: WIDTH
+    });
+
+    it('initializes correctly with canvas passed in as argument', () => {
```

Comment:
```suggestion
    it('initializes correctly with canvas passed in as argument', function() {
```

Reply:
Sorry about that. Thanks for your changes.